gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/util/CompoundStatementResourceExtractors.java

```diff
@@ -29,6 +28,17 @@
         );
     }
 
+    public static Stream<RCMRMT030101UK04CompoundStatement> extractAllChildCompoundStatements(RCMRMT030101UK04Component02 component02) {
+        return Stream.concat(
```

Comment:
Is `Stream.concat` necessary? You could just filter out elements that don't meet `component02.hasCompoundStatement()` statement